### DISCUSSION TITLE
How to add link to external site in top navigation

### DISCUSSION THREAD

**Discussion by metbril** [Lednerb/bilberry-hugo-theme#436] [Q&A] [answered]
I want to add a link to an external site (like my GitHub profile) in the top navigation. How can I achieve this? Should I just create a page with a `link:` parameter in the front matter? Wouldn't this make the page available as a real page?

**Comment by igor-baiborodine** [comment, marked as answer]
@metbril, the [example site](https://lednerb.github.io/bilberry-hugo-theme/) contains the page with redirection to GitHub: https://github.com/Lednerb/bilberry-hugo-theme/blob/master/exampleSite/content/page/github.md

![image](https://user-images.githubusercontent.com/1027701/198648719-e3798de1-9b8f-4e68-9d71-8493efcf82e4.png)

**Comment by metbril** [comment]
Thanks for the quick reply. I had already tested this. And noticed that the path `/github/` is available. When I enter it in the browser, like `https://example.org/github/` then the page is opened shortly and then redirected to the specified link.

Unfortunately bilberry used a non-standard way of Hugo menus, like the example in the docs:

https://gohugo.io/content-management/menus/#add-non-content-entries-to-a-menu

I'll give this some more thought....
